pkg/trace/exporter.go

```diff
@@ -0,0 +1,41 @@
+package trace
+
+import (
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/jaeger"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
+)
+
+// NewExporter returns a Jaeger Exporter
+func newExporter(url string) (*jaeger.Exporter, error) {
+	endpoint := jaeger.WithEndpoint(url)
+	collector := jaeger.WithCollectorEndpoint(endpoint)
+	return jaeger.New(collector)
+}
+
+func newResource(version string) *resource.Resource {
+	r, _ := resource.Merge(
+		resource.Default(),
+		resource.NewWithAttributes(
+			semconv.SchemaURL,
+			semconv.ServiceNameKey.String("authorino"),
+			semconv.ServiceVersionKey.String(version),
+			attribute.String("environment", "demo"),
```

Comment:
```suggestion
```

We don’t need this fixed attribute of course. Instead, maybe in the future we could add another optional config called, e.g., `observability-service-seed`. Users could use it, e.g., to distinguish which Authorino pod is sending the trace.